### PR TITLE
Added 'chromeOptions' from local environment variable

### PR DESCRIPTION
**Pull request by jmcameron**
When we run 'yarn test:ends' on our local computer, we each may need different options in the 'chromeOptions' part of the file.  You may need to run 'headless'; but some may not want to do that.   This mod allows you to specify an environment variable:

   CHROME_OPTIONS

which will be added to the protractor 'chromeOptions'.   For example if you want to run 'ends' headless, define:

   export CHROME_OPTIONS="--headless"

Depending on your local environment, you may need other options depending on whether you are running Chrome or Chromium.  Other options that can be used for Chromium are:
	'--no-sandbox',
	'--disable-dev-shm-usage',
	'--remote-debugging-port=9222',
	'--disable-extensions'

To find out more about these options, try searching for them on Google.

NOTE: You should be able to specify multiple options here (space separated), but I have not actually tested that.

Please make sure this works (without defining CHROME_OPTIONS) on your local computers.

### CODE REVIEW

File: protractor.conf.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ const config = {
   capabilities : {
     browserName : 'chrome',
     chromeOptions : {
-      args : ['--disable-gpu', '--window-size=1280,1024'],
+      args : ['--disable-gpu', '--window-size=1280,1024', process.env.CHROME_OPTIONS],
```
Comment (jniles) [inline protractor.conf.js]: This is a good idea, however, it doesn't work on my machine.  I tweaked it a bit and this is what works:
```js
const config = { 
  // all the params
};

if (process.env.CHROME_OPTIONS) {
  config.capabilities.chromeOptions.args.push(process.env.CHROME_OPTIONS);
}
```

**Review by jniles — CHANGES_REQUESTED** [review]
@jmcameron this doesn't work out of the box without `CHROME_OPTIONS` defined. I've proposed the fix.

File: protractor.conf.js (hunk 1 above)
Comment (jmcameron) [reply protractor.conf.js]: Good catch.  Should have tried it without an environment variable myself!

**Comment by jmcameron** [comment]
bors try

**Review by lomamech — COMMENTED** [review]
When I did the E2E test with Protractor I got the following error message
image

File: protractor.conf.js (hunk 1 above)
Comment (jmcameron) [reply protractor.conf.js]: Done

**Comment by jniles** [comment]
bors try

**Comment by jniles** [comment]
@lomamech can you re-test this?

**Comment by jniles** [comment]
Or @mbayopanda .  I just would like to make sure this is not windows specific.

File: protractor.conf.js [hunk 2]
```diff
@@ -78,5 +78,10 @@ if (process.env.CI) {
   delete config.plugins;
 }
 
+// Add custom Chrome options for local environments
+if (!process.env.CI && process.env.CHROME_OPTIONS) {
```
Comment (mbayopanda) [inline protractor.conf.js:82]: The test e2e works fine, I would like just to know what kind of values `CHROME_OPTIONS` can take, also if it is possible, add them in the comment as an example

**Review by mbayopanda — APPROVED** [review]
LGTM 👍 

I just left a non-blocking comment about the type of use of `CHROME_OPTIONS`

File: protractor.conf.js (hunk 2 above)
Comment (jmcameron) [reply protractor.conf.js:82]: @mbayopanda I added a little more info on the PR description at the top. 
Thanks for testing it!

**Review by jniles — APPROVED** [review]

**Comment by jniles** [comment]
This is excellent, thank you.

bors r+

**Comment by jniles** [comment]
This works for me and @mbayopanda .

bors r+

**Pull request merged**